src/App.tsx

```diff
@@ -157,6 +161,9 @@ const App: React.FC = () => {
             <Route path="/collectibles">
               <Redirect to="/nfts" />
             </Route>
+            <Route path="/profile">
+              <Redirect to={`${nftsBaseUrl}/profile/${account?.toLowerCase() || nullProfileAccountAddress}`} />
```

Comment:
Set up redirect to users' profile (if connected), and a null address if not

Reply:
This is not necessary. Simply make the address optional and handle at the view level.
```
<Route path="/profile/:address?" />
```